### PR TITLE
Ignore Visual Studio generated files specific to user workspace

### PR DESCRIPTION
**Pull request by mrotteveel**
I have made the pull request against B3_0_Release. I think it should also be applied to master. Let me know if you need another pull request.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@ config.status
 configure
 libtool
 *.log
+builds/win32/*/*.sdf
+builds/win32/*/*.VC.db
+builds/win32/*/*.VC.opendb
```
Comment (hvlad) [inline .gitignore]: We already have .gitignore at builds/win32. I think such changes shouild be there.

Comment (mrotteveel) [inline .gitignore]: I hadn't noticed that one. I will create a new pull request this weekend.

Comment (hvlad) [inline .gitignore]: Mark, i can do necessary changes, if you wish

Comment (mrotteveel) [inline .gitignore]: If you can do that, that would be great. Thanks!

Comment (hvlad) [inline .gitignore]: Done

**Pull request closed without merging**